server/pyproject.toml

```diff
@@ -0,0 +1,12 @@
+[tool.black]
+line-length = 80
```

Comment:
I agree with your reservations, we should probably make this 88 (black default) or higher. 80 is kind of short.